styles/abstracts/_theme.scss

```diff
@@ -0,0 +1,48 @@
+$themes: (
+  'light-mode': (
+    'white': #ffffff,
+    'black': #000000,
+    'text-color': #000000,
+    'text-color-inverse': #ffffff,
+    'k-accent': #ff7ac3,
+    'k-accentlight': #ffe5f3,
+    'k-accentlight2': #fff0f8,
+    'k-green': #04af00,
+    'k-red': #ff5757,
+    'k-grey': #999999,
+    'k-hovergrey': #6b6b6b,
+    'k-blue': #6188e7,
+    'k-shade': #cccccc,
+    'primary-shadow': 4px 4px $black,
+  ),
+  'dark-mode': (
+    'white': #ffffff,
+    'black': #000000,
+    'text-color': #ffffff,
+    'text-color-inverse': #000000,
+    'k-accent': #191718,
+    'k-accentlight': #ffe5f3,
+    'k-accentlight2': #363234,
+    'k-green': #04af00,
+    'k-red': #ff5757,
+    'k-grey': #999999,
+    'k-hovergrey': #6b6b6b,
+    'k-blue': #6188e7,
+    'k-shade': #cccccc,
+    'primary-shadow': 4px 4px $white,
+  ),
+);
+
+@mixin ktheme() {
```

Comment:
Is there any advantage to using the scss `minxin` compare to variable I mentioned above? I found it's not that easy to use mixin in the style file and would introduce more dumplicated code, like 
```@include ktheme() { theme(xxx); }```